content/projects/jenkins-operator/index.adoc

```diff
@@ -0,0 +1,42 @@
+---
+layout: project
+title: "Jenkins Operator"
+section: projects
+tags:
+- kubernetes
+- infrastructure
+links:
```

Comment:
Would be also great to add OpenGraph metadata here. Just to be able to link this page in social media easily

Reply:
I believe this is what you're requesting. Thanks